src/Components/Browser.JS/package.json

```diff
@@ -11,8 +11,7 @@
     "test": "jest"
   },
   "devDependencies": {
-    "@aspnet/signalr": "^1.0.0",
-    "@aspnet/signalr-protocol-msgpack": "^1.0.0",
```

Comment:
should `@aspnet/signalr` still be here?

Reply:
We get it transitively from signalr-protocol-msgpack